functional-test/src/test/groovy/org/rundeck/tests/functional/api/job/JobScheduledSpec.groovy

```diff
@@ -28,13 +28,14 @@ class JobScheduledSpec extends BaseContainer {
             def nh = upDate.hour
             def nm = upDate.minute
             def ns = upDate.second
+            def randomUUID = UUID.randomUUID().toString()
 
             def xml = """
                 <joblist>
                    <job>
                       <name>scheduled job</name>
                       <group>api-test/job-run-scheduled</group>
-                      <uuid>api-test-job-run-scheduled</uuid>
+                      <uuid>${randomUUID}</uuid>
```

Comment:
@gschueler @mrdubr,  I am working on this JobSchedule groovy test (mostly practicing/ learning), and initially it had a hard coded UUID. When I ran the test, it worked the first time but failed on subsequent runs, showing an error that the same UUID cannot be used again. To resolve this, I replaced the hard coded UUID  with a random UUID. After making this changes, the tests runs successfully every time as the UUID is unique for each run.I also confirmed that the tests are passing in the localhost. 
Since this is my first time working on this, I wanted to check if there is any cache or underlying mechanism that I should clear or reset when running these tests? Or Is there anything else I might be missing that could cause conflicts during subsequent test runs?

Sorry accidentally I have pushed package-lock.json. I will delete this file